client/my-sites/customer-home/locations/secondary/learn-grow/index.jsx

```diff
@@ -42,7 +45,7 @@ const LearnGrow = ( { cards, trackCards } ) => {
 	if ( ! cards || ! cards.length ) {
 		return null;
 	}
-
+	console.dir( cardComponents );
```

Comment:
Did you mean to leave this console statement in here?

Reply:
Nope, I removed it!